fighters/common/src/function_hooks/sound.rs

```diff
@@ -21,9 +22,43 @@ unsafe fn PLAY_SE_hook(lua_state: u64) {
     original!()(lua_state);
 }
 
+#[skyline::hook(offset = 0x4cf6a0)]
+unsafe fn soundmodule__play_se_hook(sound_module: u64, se: smash::phx::Hash40, arg2: bool, arg3: bool, arg4: bool, arg5: bool, se_type: smash::app::enSEType) -> u64 {
+    let handle = original!()(sound_module, se, arg2, arg3, arg4, arg5, se_type);
+    let boma = *(sound_module as *mut *mut BattleObjectModuleAccessor).add(1);
+    if !utils::se::se_list().contains(&se.hash) {
```

Comment:
this not being a `HashSet` is a crime against humanity

Reply:
done